handlers.go

```diff
@@ -125,3 +126,25 @@ type channelHandler struct{ ch *Channel }
 func (c channelHandler) Handle(ctx context.Context, call *InboundCall) {
 	c.ch.GetSubChannel(call.ServiceName()).handler.Handle(ctx, call)
 }
+
+// channelHandlerWithNative is a Handler that wraps a Channel and delegates requests
+// to SubChannels if the inbound call's service and method name are configured to
+// ignore passed-in thirdparty handlers.
+type userHandlerWithIgnore struct {
+	ch          *Channel
```

Comment:
instead of taking the channel, suggest taking a `Handler` for the ignored cases, and that handler can be set to `channelHandler{ch}` at the caller (same as if no ignores were set) vs doing a subchannel lookup ourselves